MANIFEST.in

```diff
@@ -0,0 +1,10 @@
+include README.rst
+include LICENSE.txt
+include CHANGELOG.md
```

Comment:
Make sure the license file is definitely included. Include the README and CHANGELOG as well for good measure.